src/transformers/processing_utils.py

```diff
@@ -993,6 +990,59 @@ def apply_chat_template(
         )
 
 
+def _check_reversed_images_text_for_vlms(images, text):
```

Comment:
This behaviour should be tested

Reply:
@amyeroberts since this function is in processing_utils, where would the best place be to test it? Or should we create model-specific tests for all models using it?

Reply:
If there isn't a module for testing utils in processing_utils then we should add one!